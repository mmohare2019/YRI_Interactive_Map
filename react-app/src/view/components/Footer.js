import React from "react";

function Footer() {
    return (
        <nav class="navbar fixed-bottom navbar-light bg-light" className="footer">
            <a class="navbar-link" href="/message">Submit request</a>
        </nav>
    );
}

export default Footer;