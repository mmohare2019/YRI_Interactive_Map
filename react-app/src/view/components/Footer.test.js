import TestRenderer from "react-test-renderer";
import Footer from "./Footer";
import React from "react";
import renderer from 'react-test-renderer';

describe("Footer render tests", () => {
    test("render smoke test", () => {
        const testRenderer = TestRenderer.create(<Footer/>)
    })
});

test("header test", function () {
    const component = renderer.create(<Footer/>).toJSON();
    expect(component).toMatchSnapshot(); 
});