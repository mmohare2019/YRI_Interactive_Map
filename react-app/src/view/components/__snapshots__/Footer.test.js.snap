// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header test 1`] = `
<nav
  class="navbar fixed-bottom navbar-light bg-light"
  className="footer"
>
  <a
    className="nav-item nav-link"
    href="/message"
  >
    Submit request
  </a>
</nav>
`;
